### ISSUE TITLE
Commentary

### ISSUE THREAD

**Issue by canweriotnow** [exercism/xelisp#18]
Since elisp packages have a very structured and opinionated format, requiring a `;;; Commentary:` section, it might be good to add some language-specific guidance for each exercise here. I'll update with a running checklist of exercises that have/need commentary, but I figure if Emacs says do it, and it could be helpful to users, who am I to argue?

**Comment by yurrriq** [comment]
It might be good to have a description of how to set up [flycheck](https://github.com/flycheck/flycheck) too!

**Comment by canweriotnow** [comment]
Yes :+1: to setting up Flycheck. I think I suggest using emacs-prelude for new users in my help guide, which comes with flycheck installed and configured, but a separate guide would be good.
